worker/include/RTC/RTCP/SenderExtendedReport.hpp

```diff
@@ -0,0 +1,222 @@
+#ifndef MS_RTC_RTCP_SENDER_EXTENDED_REPORT_HPP
+#define MS_RTC_RTCP_SENDER_EXTENDED_REPORT_HPP
+
+#include "common.hpp"
+#include "Logger.hpp"
+#include "Utils.hpp"
+#include "RTC/RTCP/Packet.hpp"
+#include <vector>
+
+namespace RTC
+{
+	namespace RTCP
+	{
+		class SenderExtendedReport
+		{
+		public:
+			struct Header
+			{
+				uint8_t blockType;
+				uint8_t reserved;
+				uint16_t blockLength;
+				uint32_t ssrc;
+				uint32_t lrr;
+				uint32_t dlrr;
+			};
```

Comment:
This header does not represent a DLRR Report Block as per the [spec](https://tools.ietf.org/html/rfc3611#section-4.5)

The bock is defined by:

- blockType = 5
- reserved = 0
- blockLength = (depends on number of sub-blocks)
- N * sub-blocks.

The sub-block is defined by:
- ssrc
- lrr
- dlrr